package/src/application/LabelingTool/LabelingTool.styles.js

```diff
@@ -34,8 +34,12 @@ const GlobalStyle = createGlobalStyle`${css`
     background: ${COLORS.GRAYSCALE_1};
   }
 
-  scrollbar-color: ${COLORS.PRIMARY_3} ${COLORS.GRAYSCALE_1};
-  scrollbar-width: thin;
+  @supports not selector(::-webkit-scrollbar) {
+    * {
```

Comment:
do we need additional `*` block? now it's `*` in `*`

Reply:
no, fixed